source/MapDetailPanel.cpp

```diff
@@ -838,7 +838,10 @@ void MapDetailPanel::DrawInfo()
 		text.SetWrapWidth(WIDTH - 20);
 		text.Wrap(selectedPlanet->Description());
 		text.Draw(Point(Screen::Right() - X_OFFSET - WIDTH, Screen::Top() + 20), medium);
-	}
+
+		selectedSystemOffset = -150;
+	} else
+		selectedSystemOffset = 0;
```

Comment:
Since the code to reset this variable to zero is in the MapDetailPanel, switching to another map panel while the selected system UI is offset will keep it offset until returning to the ports panel.